exe-common/src/config.rs

```diff
@@ -342,6 +342,28 @@ pub mod net {
             }
         }
 
+        pub fn obft_testnet() -> Self {
```

Comment:
We don't want to embed obft testnet in the library

Reply:
Makes sense

Reply:
Will remove then. Along with the weird hack for genesis.

Reply:
It seems like we don't need obft-testnet anymore either, since staging seems to be working fine with OBFT. So yes. No reason to keep it.